sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml

```diff
@@ -0,0 +1,52 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+        - "COVID19_ALL.h5ad"
+dataset_wise:
+    author: "Ren, Xianwen"
+    default_embedding: "X_tsne"
+    doi_preprint:
+    doi_journal: "10.1016/j.cell.2021.01.053"
+    download_url_data: "https://drive.google.com/file/d/1IwWcn4W-YKgNbz4DpNweM2cKxlx1hbM0/view"
+    download_url_meta: "None"
```

Comment:
@lauradmartens was this "None" introduced by the CLI? If so we need to take it out, this field should be empty if there is no metadata URL.